pages/settings/devices.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <settings-page page-title="settings.devices.devices">
+    <template #actions>
+      <v-btn
+        v-if="devices"
+        color="error"
+        class="ml-a"
+        @click="deleteAllDevices"
+      >
+        {{ $t('settings.devices.deleteAll') }}
+      </v-btn>
+      <v-btn
+        ref="noreferrer noopener"
+        href="https://jellyfin.org/docs/general/server/devices.html"
+        target="_blank"
+      >
+        {{ $t('settings.help') }}
+      </v-btn>
+    </template>
+    <template #content>
+      <v-col>
+        <v-data-table
+          :headers="headers"
+          :items="devices"
+          @click:row="setSelectedDevice"
+        >
+          <!-- eslint-disable-next-line vue/valid-v-slot -->
+          <template #item.DateLastActivity="{ item }">
+            <p class="text-capitalize-first-letter mb-0">
+              {{
+                $dateFns.formatRelative(
+                  $dateFns.parseJSON(item.DateLastActivity),
+                  new Date(),
+                  {
+                    locale: getDfnsLocale()
+                  }
```

Comment:
```suggestion
                  {
                    locale: this.$i18n.locale
                  }
```

That mixin is going to be deprecated in #633. Now i18n locales matches datefns ones, so this will work.

Reply:
I tried just using this but it wasn't working, I'll take another look